frontend/src/app/search.tsx

```diff
@@ -17,8 +18,15 @@ export default class Search extends Component {
     sortKey: '',
     pageOffset: 1,
     pageCount: 5,
-    initialLoad: true
+    initialLoad: true,
+    value: false,
+    show: false,
+    count: 0,
+    flag: false
```

Comment:
Can we please name these variables more specifically? Instead of `value`, perhaps `checkedForDelete` (or something along those lines). Same thing for `show` - how about `deleteButtonVisible`? Same for `count` and `flag` - whatever's appropriate.

Reply:
Acknowledged.
Thank you for catching this. I will make the changes to the names of the variables specific to their functionality.